docs/platform.rst

```diff
@@ -82,6 +84,35 @@ The typical things an engine needs to handle are:
   same base class, apps can call methods on the engines to for example create UIs. It is up to each
   engine to implement these methods so that they work nicely inside the host application.
 
+- A startup interface for related DCC applications.
+
```

Comment:
If this feels like too much for this section, maybe a separate section somewhere that outlines the whole thing that can be linked to from here. I have a feeling that for folks new to TK, they're going to need a little more context.